dashboard/app/controllers/plc/content_creator_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Plc::ContentCreatorController < ApplicationController
-  before_filter :require_levelbuilder_mode
-
   def show_courses_and_modules
+    authorize! :read, Plc::Course
```

Comment:
Sctually I see that you don't have any specific permissions set for Plc::Course, so admin can read and manage and this is effectively the same thing.